__tests__/services/CleanupService.spec.ts

```diff
@@ -68,10 +68,10 @@ const wildCardGreyGit: Expression = {
   id: '4',
   listType: ListType.GREY,
   storeId: 'default',
-}
+};
 
 const whiteListAllExceptTwitter: Expression = {
-  expression: '/^((?!twitter\.com).)+$/',
+  expression: '/^((?!twitter.com).)+$/',
```

Comment:
This is a regex expression - the backslash needs to stay.

Reply:
https://eslint.org/docs/rules/no-useless-escape

> Escaping non-special characters in strings, template literals, and regular expressions doesn't have any effect

The tests seem to pass just fine, so I think we're ok here

Reply:
I think the check may be treating the expression as a string instead of its intended regex...
I suppose.
Isn't a dot a special character?  It stands for any one character.
  It may just look weird because if we're manually reading this the regex would mean that 'twitterxcom' would match...

Reply:
https://stackoverflow.com/questions/47277133/disable-unnecessary-escape-character-no-useless-escape

For the most part it seems safe to remove the `\`, but if we need to, we can use `[.]` instead.

Reply:
Because we accept regex and string in the expression, eslint doesn't know that.  I think this line should be the exception to the no-useless-escape eslint.

Reply:
> https://stackoverflow.com/questions/47277133/disable-unnecessary-escape-character-no-useless-escape
> 
> For the most part it seems safe to remove the `\`, but if we need to, we can use `[.]` instead.

I would agree with this had the \ be within the `[]`, but there wasn't any.

Reply:
I guess for testing purposes and visual look the [.] should be done then.  Seems a bit excessive for an extra character compared to `\.` but if that will make eslint not warn, then sure.